docs/source/user_guide/config.rst

```diff
@@ -662,60 +662,62 @@ property.
-| `plugins`                 | N/A                                    | `{}`                        | A dict of plugin configurations. See :ref:`this section <configuring-plugins>` for        |
-|                           |                                        |                             | details.                                                                                  |
-+---------------------------+----------------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
++----------------------------+-----------------------------------------+---------------+--------------------------------------------------------------------------------------------+
+| Config field               | Environment variable                    | Default value | Description                                                                                |
++============================+=========================================+===============+============================================================================================+
+| `color_by`                 | `FIFTYONE_APP_COLOR_BY`                 | `"field"`     | Whether to color labels by their field name (`"field"`), `label` value (`"label"`), or     |
+|                            |                                         |               | render each instance ID/trajectory index (`"instance"`).                                   |
++----------------------------+-----------------------------------------+---------------+--------------------------------------------------------------------------------------------+
+| `color_pool`               | `FIFTYONE_APP_COLOR_POOL`               | See below     | A list of browser supported color strings from which the App should draw from when         |
+|                            |                                         |               | drawing labels (e.g., object bounding boxes).                                              |
++----------------------------+-----------------------------------------+---------------+--------------------------------------------------------------------------------------------+
+| `colorscale`               | `FIFTYONE_APP_COLORSCALE`               | `"viridis"`   | The colorscale to use when rendering heatmaps in the App. See                              |
+|                            |                                         |               | :ref:`this section <heatmaps>` for more details.                                           |
++----------------------------+-----------------------------------------+---------------+--------------------------------------------------------------------------------------------+
+| `disable_frame_filtering`  | `FIFTYONE_APP_DISABLE_FRAME_FILTERING`  | `False`       | Whether to disable frame filtering for video datasets in the App's grid view.              |
```

Comment:
FYI- I updated the description here to prepare for the intended state where this setting only affects the grid view